docs(glossary): french translation (a to c)

cexbrayat opened the pull request:
Thank you for submitting this PR...

**Please check if the PR fulfills these requirements**
- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit-message-format
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)


**Please provide the issue number?**

issue #4

**Did you finish translating the whole issue?**
- [X] no (pr_state: WIP)
- [ ] yes (pr_state: PLEASE_REVIEW)

**What kind of change does this PR introduce?** (check one with "x")
- [X] Translation
- [ ] Other... Please describe:

manekinekko commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -26,18 +36,29 @@ include _util-fns
 :marked
   ## Attribute Directive
+
+  ## Directive Attribut
```
Que penses-tu de `Directives d'Attribut` plutôt ?

cexbrayat replied:
Ca me va, je corrige 👍

manekinekko commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -48,42 +69,59 @@ include _util-fns
     The barrel itself is a module file that re-exports *selected* exports of other modules.
-    
+
+    Un baril est une façon de *regrouper les exports* de différents modules en un seul et plus pratique module.
```
Suggestion : `...en un seul module, ce qui est très pratique.`

cbalit replied:
je dirais même plus : 

> ...en un seul module, ce qui est plus pratique

manekinekko commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -94,10 +132,16 @@ include _util-fns
     This more rare usage should be clear in context.
 
+    Peut aussi faire référence à un lien dans l'[Injection de Dépendances](#dependency-injection)
+    entre une "clé" et un [fournisseur](#provider) (_provider_) de dépendance.
```
J'ajouterais aussi `... "clé" (_Token_) ...`

Gillespie59 commented:
pas de remarques supplémentaires

ctesniere commented on public/docs/ts/latest/glossary.jade:
```diff
@@ -48,42 +69,59 @@ include _util-fns
   :marked
     The Angular [scoped packages](#scoped-package) each have a barrel named `index`.
+
+    Les [packages](#scoped-package) Angular ont chacun un baril nommé `index`.
```
Pour être cohérents avec le reste, il faudrait ajouter : `(_scoped-package_)` ?

cexbrayat replied:
J'avoue que j'étais sec sur comment traduire scoped packages en FR... Je laisse comme ça ou tu as une idée ?

manekinekko commented:
@cexbrayat des news ?

cexbrayat commented:
Nope pas eu le temps cette semaine

cexbrayat commented:
@manekinekko Fixes, rebased and squashed. Can be merged it it looks good, I'll open another PR for the following terms in the glossary (will be easier to review).
